spec/features/visitor_can_view_categories_page_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe 'Visitor can view category pages', type: :feature do
+  scenario 'visitor clicks link to category page from homepage' do
+    category_1 = FactoryGirl.create(:category)
+    category_2 = FactoryGirl.create(:category)
```

Comment:
factory girl allows you to do something like: 
 category_1, category_2 = create_list(:category, 2)

In order to maintain consistency throughout all of our tests, we should decide on one together :)